src/transaction/log_page_buffer.c

```diff
@@ -310,6 +310,13 @@ static void logpb_set_unavailable_archive (THREAD_ENTRY * thread_p, int arv_num)
 static void logpb_dismount_log_archive (THREAD_ENTRY * thread_p);
 static bool logpb_is_archive_available (THREAD_ENTRY * thread_p, int arv_num);
 static void logpb_archive_active_log (THREAD_ENTRY * thread_p);
+#if defined (SERVER_MODE)
+static void logpb_assert_correctness_read_page_from_file_or_page_server (LOG_PAGEID pageid,
```

Comment:
This name really screams to be shortened :)). What about just `logpb_verify_page_read`?

Reply:
Done